edxapp/Dockerfile

```diff
@@ -23,15 +23,19 @@ VOLUME /openedx/data
 WORKDIR /openedx/edx-platform
 
 ## Checkout edx-platform code
-RUN git clone https://github.com/edx/edx-platform.git --branch open-release/ginkgo.master --depth 1 .
+ARG EDX_PLATFORM_REPOSITORY=https://github.com/edx/edx-platform.git
+ARG EDX_PLATFORM_VERSION=open-release/ginkgo.master
```

Comment:
And try out different branches of the repo instead of hardcoding it to open-release/ginkgo.master